### ISSUE TITLE
Change data capture tool?

### ISSUE THREAD

**Issue by wudihero2** [rewrite-bigdata-in-rust/RBIR#35]
Hi, does anyone know if Rust has a change data capture tool? I think in real time scenario, it would be an import part that need to be added to RBIR. Thanks.

**Comment by sdf-jkl** [comment]
Hi @wudihero2 I came across https://supermetal.io/

**Comment by wudihero2** [comment]
Hi @sdf-jkl, it looks nice! Just curious, is it open source?

**Comment by sdf-jkl** [comment]
I don't think so. https://github.com/datazip-inc/olake is open source and does CDC

**Comment by wudihero2** [comment]
Thanks👍👍👍 
Other than Debezium, I haven’t seen many options for DB2 CDC. Have you seen any others?

**Comment by sdf-jkl** [comment]
I haven't come across any others. I'll let you know if I do.
